docs/platform/webcontainers/browser-config.md

```diff
@@ -57,6 +57,38 @@ Note that the list of blocked domains might look different for you. In particula
 
 :::
 
+## Firefox: enabling Service Workers {#firefox-service-workers}
+
+When Firefox’s Enhanced Tracking Protection is configured in “Custom” mode with the cookie blocking option set to “All cross-site cookies”, Firefox will block the Service Workers used by WebContainers.
+
+You can work around this issue in a couple ways:
+
+1. Use the “Strict” mode instead of “Custom” mode. It’s slightly more permissive than “Custom” mode with very strict options, so if you’d rather not, you can…
```

Comment:
```suggestion
1. Use the “Strict” mode instead of “Custom” mode. It’s important to note that it's slightly more permissive than “Custom” mode with very strict options. If this is not something you'd like to allow, you can...
```